cou/apps/auxiliary.py

```diff
@@ -177,3 +177,8 @@ class MysqlInnodbClusterApplication(OpenStackAuxiliaryApplication):
     # NOTE(agileshaw): holding 'mysql-server-core-8.0' package prevents undesired
     # mysqld processes from restarting, which lead to outages
     packages_to_hold: Optional[list] = ["mysql-server-core-8.0"]
+
+
+@AppFactory.register_application(["ceph-osd"])
+class CephOSD(OpenStackAuxiliaryApplication):
+    """Application for ceph-osd."""
```

Comment:
It's not enough to add `ceph-osd` to `register_application` for `OpenStackAuxiliaryApplication`?

Reply:
Yes and no. Ceph-osd will need special upgrade steps for sure and my concern is that we forget to unregister on  `OpenStackAuxiliaryApplication`

Reply:
ok then I think you could add a comment indicating that although it's empty for now, this is the right place for the new class as it will need overrides later on